generator/internal/scaffold_generator.cc

```diff
@@ -942,22 +942,23 @@ void GenerateQuickstartSkeleton(
 // limitations under the License.
 
 //! [all]
-#include "google/cloud/$library$/ EDIT HERE .h"
-#include "google/cloud/project.h"
+#include "google/cloud/$library$/vN/ EDIT HERE _client.h"
+#include "google/cloud/location.h"
 #include <iostream>
 
 int main(int argc, char* argv[]) try {
-  if (argc != 2) {
-    std::cerr << "Usage: " << argv[0] << " project-id\n";
+  if (argc != 3) {
+    std::cerr << "Usage: " << argv[0] << " project-id location-id\n";
     return 1;
   }
 
-  namespace $library$ = ::google::cloud::$library$;
+  auto const location = google::cloud::Location(argv[1], argv[2]);
+
+  namespace $library$ = ::google::cloud::$library$_v/*EDIT HERE*/;
```

Comment:
same here.

Reply:
This one was more difficult.  `$service_subdirectory$` doesn't apply in this context, so I had to add a `$product_namespace$` to the scaffolding vars.